packages/connect/src/http_client.ts

```diff
@@ -38,34 +44,37 @@ export class HttpClient implements Client {
     }
     /**
      * Retrieve token pair info from the API
-     * @param   request     A TokenPairsRequest instance describing specific token information
-     *                      to retrieve
+     * @param   requestOpts     An optional (TokenPairsRequestOpts & PagedRequestOpts) instance describing token information
+     *                          to retrieve with page information, defaults to { page: 0, per_page: 100 }
```

Comment:
`Request options specifying the specific token pairs to retrieve as well as page options (page index and/or number of items per page to fetch). Defaults to { page: 0, per_page: 100 }.`

Reply:
Don't think we need to put the type in the description. Big chance they become out of sync in the future.